docs/markdown/tutorial/training.md

```diff
@@ -1,173 +1,437 @@
 # Tutorial for Training Library
+Hyperpose python training library provides a one-step yet flexible platform for developing pose estimation models.
+
+Based on the intented usage, there are two kinds of users of the pose estimation development platform:
+- **1.Users aim to adapt the existing algorithms to specific deployment scenario.**
+
+  **Example:** Search for the pose estimation model architecture with best accuracy conditioned on the limited hardware resources.
+
+- **2.Users aim to design and polish their own pose estimation algorithms.**
+
+  **Example:** Explore new pose estimation algorithms with the help of existing dataset generation pipeline and model architectures. 
+
+To meet the requirements of two kinds of users, Hyperpose provides both abundant High-level APIs with integrated pipelines (for first kind of users) and fine-grained APIs with in-depth customisation (for second kidn of users). 
```

Comment:
- "two kinds of users" -> "2 kinds of user requirements"
- "Hyperpose" -> "HyperPose"
- "abundant" -> "rich"
- "High-level" -> "high-level"
- "for second kidn of users" -> "for the second kind of requirement"

Reply:
fixed.